### DISCUSSION TITLE
生物大危机！

### DISCUSSION THREAD

**Discussion by keerdun** [share121/inter-knot#663] [其他]
![1723226396189](https://github.com/user-attachments/assets/83ea048e-1672-4ffd-a7ba-a94053d51a79)
  各位，最近我发现家里出现了许多黑色的小蛤蟆，大概和一个大拇指肚差不多大，而且极为泛滥，但往年都没有，这是怎么回事？

**Comment by Micawber-jock** [comment]
油炸指定豪赤

**Comment by lllliufeng** [comment]
黑色的小蛤蟆不要扔，裹上鸡蛋液，粘上面包糠，下锅炸至金黄酥脆控油捞出，老人小孩都爱吃，隔壁小孩都馋哭了！！！

**Comment by Hongxuli2006** [comment]
没开玩笑，这是什么东西啊

**Comment by tiancaitu** [comment]
好歹也拍张图片发上来呀
